### PR TITLE
Fix OpenAPI spec generation action

### PR DESCRIPTION
**Pull request by shangyian**
### Summary

Now that we've moved the relevant script to live under `datajunction-server/scripts/generate-openapi.py`, we need to change this action to reference the right location to run the script.

### Test Plan

- [ ] PR has an associated issue: #
- [x] `make check` passes
- [x] `make test` shows 100% unit test coverage

### CODE REVIEW

**Comment by samredai** [comment]
Thanks for catching this. Would it be better to make the change below so that the spec is still placed in the root directory?

```diff
      - name: Generate OpenAPI Spec
-        run: ./scripts/generate-openapi.py -o openapi.json
+        run: ./scripts/generate-openapi.py -o ../openapi.json
```

**Review by samredai — APPROVED** [review]

**Comment by shangyian** [comment]
@samredai I noticed that this is used later to generate the markdown file in the docs. I guess that could make sense as a part of this step too so that it's entirely automated.
> `widdershins openapi.json -o docs/content/0.1.0/docs/developers/the-datajunction-api-specification.md --code=true --omitBody=true --summary=true`

**Comment by samredai** [comment]
> @samredai I noticed that this is used later to generate the markdown file in the docs. I guess that could make sense as a part of this step too so that it's entirely automated. What do you think?
> 
> > `widdershins openapi.json -o docs/content/0.1.0/docs/developers/the-datajunction-api-specification.md --code=true --omitBody=true --summary=true`

Great idea, that makes sense to me!

File: docker-compose.yml [hunk 1]
```diff
@@ -12,8 +12,6 @@ services:
   dj:
     container_name: dj
     stdin_open: true
-    env_file:
```
Comment (shangyian) [inline docker-compose.yml:15]: @samredai removing it here

**Pull request merged**